pkg/version/version.go

```diff
@@ -55,11 +55,17 @@ func String() string {
 }
 
 func GetBuild() *Build {
+	var b string
+	if BuildNumber == "" {
+		b = "0"
```

Comment:
good point BuildNumber is set here: https://github.com/vmware/vic/blob/master/Makefile#L57 based on env var which usually comes from drone